blacklistfilter/blacklist_downloader/bl_downloader.py.in

```diff
@@ -147,12 +183,12 @@ class Blacklist:
 
                     logger.info('Updated {}: {}'.format(type(self).__name__, self.name))
 
-                if not new_entities:
+                if new_entities is None:
                     logger.warning('{}, {}: No valid entities found'.format(type(self).__name__, self.name))
```

Comment:
The preivous version (`not new_entities`) was actually better, as it matches `None` as well as empty list/set - which is what we want to warn about.

Reply:
Fixed